Cargo.toml

```diff
@@ -643,7 +643,8 @@ backoff = { version = "0.4" }
 backtrace = { version = "0.3.71" }
 binary-merkle-tree = { path = "substrate/utils/binary-merkle-tree", default-features = false }
 bincode = { version = "1.3.3" }
-bip39 = { version = "2.0.0" }
+# personal fork here as workaround for: https://github.com/rust-bitcoin/rust-bip39/pull/64
+bip39 = { package = "parity-bip39", version = "2.0.1", default-features = false }
```

Comment:
Can we use `bip39 = { workspace = true }` here ? I see the https://github.com/rust-bitcoin/rust-bip39/pull/64 was merged

Reply:
Unfortunately, it can't.
There's a following issue https://github.com/rust-bitcoin/rust-bip39/pull/69
I've pinged the maintainer, hope it can be merged soon.